garage/contrib/exp/agents/agent.py

```diff
@@ -0,0 +1,43 @@
+import abc
+
+
+class Agent(abc.ABC):
+    @abc.abstractmethod
+    def get_actions(self, obs):
+        """Get actions given a batch of observations.
+
+        Args:
+            obs: Tensor(batch_size, observation_dim)
```

Comment:
hmm what is a `Tensor`? Is it an `np.ndarray`? Does it conform to some `akro.space`?